### ISSUE TITLE
Blog page should be brought in line with rest of site visual identity

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#134]
Will be addressed in design

**Comment by vz3** [comment]
@mollieru Would you mind taking a swing at this? The blog page is basically entirely undesigned right now. I'm generally a big fan of a river format without previews myself, but I'll let you make that call.

…and if you find yourself interested in honing in the community/topic pages in general, all the more power to you =)

http://next.data.gov/energy/community/energy/blogs

**Comment by kvuppala** [comment]
In the original comp attached below we had the author images, but in this version we didn't include as the author pics are missing.

![energy_blog_comp](https://f.cloud.github.com/assets/1738764/1601084/be034a3a-5381-11e3-8e7f-80add069fea8.png)

**Comment by vz3** [comment]
This design is really far off from the rest of our visual identity. The dates in particular are an issue, and that green on white will almost certainly will not pass accessibility testing. Hoping @mollieru will be able to take a round at bringing this in line with the rest of the site.

**Comment by vz3** [comment]
@JeanneHolm Also worth noting: do we have photos for past authors? Is this something we're sure we want to highlight? I'm fine with it, there's just a lot of logistical ramifications here.

**Comment by vz3** [comment]
Changed title of this. Going to change titles on a few others to make stories more actionable.

**Comment by mollieru** [comment]
Apologies for the radio silence - yes, I'll give this a shot!

**Comment by mollieru** [comment]
So in my humble opinion, the Pinterest-tile effect would be better used on the Blog page and scrapped in the News sections.  (I'm also just wondering if these two sections could be condensed into one).  @vz3 thoughts?

**Comment by vz3** [comment]
@mollieru Definitely open to that and think it's a better use of the tile thing (though I'll go on record on hating the whole tile thing in general).

**Comment by mollieru** [comment]
Let me join you on the record. If it's possible to scrap it before the
deadline, the world would be a happy place.

On Friday, November 22, 2013, Victor Zapanta wrote:

> @mollieru https://github.com/mollieru Definitely open to that and think
> it's a better use of the tile thing (though I'll go on record on hating the
> whole tile thing in general).
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/GSA/data.gov/issues/134#issuecomment-29111583
> .

## 

## 

Mollie Ruskin
503.928.2988
mollieru@gmail.com

**Comment by ErieMeyer** [comment]
So, the good news is that this will be iterative!

I would love to land in a place where we can focus on getting the IA, communities and blog page both nailed and up on next.data.gov.

The tile approach is one that the data.gov team is also has ideas about improving, so it's definitely something we can revisit. I just don't want us to spend too much time wheel-spinning.

**Comment by mollieru** [comment]
Erie - peep the layout proposal here: https://github.com/GSA/data.gov/issues/74#issuecomment-29101923

**Comment by mollieru** [comment]
That sounds perfect. I think the IA and nab structure is the most
important. Erie - any thoughts on this approach to redesign of Nav?
https://github.com/GSA/data.gov/issues/74#issuecomment-29101923

On Friday, November 22, 2013, Erie Meyer wrote:

> So, the good news is that this will be iterative!
> 
> I would love to land in a place where we can focus on getting the IA,
> communities and blog page both nailed and up on next.data.gov.
> 
> The tile approach is one that the data.gov team is also has ideas about
> improving, so it's definitely something we can revisit. I just don't want
> us to spend too much time wheel-spinning.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/GSA/data.gov/issues/134#issuecomment-29114519
> .

## 

## 

Mollie Ruskin
503.928.2988
mollieru@gmail.com

**Comment by ErieMeyer** [comment]
I love it!

**Comment by philipashlock** [comment]
The blog pages are now the front page of the topic sections and match the layout of posts on the frontpage.

**Comment by JeanneHolm** [comment]
The blog links on the topic pages have been recently removed and should be included if those are the answer to this issue.
